### PR TITLE
fix fail in credit button

### PR DESCRIPTION
**Pull request by spekss**
fix this

### CODE REVIEW

File: js/client-mainmenu.js [hunk 1]
```diff
@@ -1095,9 +1095,9 @@
 			buf += '<li><p>[<strong>The Immortal</strong>] <small>&ndash; Development</small></p></li></ul>';
 			buf += '<h2>Retired Staff</h2>';
 			buf += '<ul><li><p><a href="http://meltsner.com/" target="_blank" class="subtle"><strong>Bill Meltsner</strong> [bmelts]</a> <small>&ndash; Development</small></p></li>';
-			buf += '<li><p><a href="https://cathyjf.com/" target="_blank" class="subtle"><strong>Cathy J. Fitzpatrick</strong> [cathyjf]</a> <small>&ndash; Development, Sysadmin</small></p></li></ul>';
+			buf += '<li><p><a href="https://cathyjf.com/" target="_blank" class="subtle"><strong>Cathy J. Fitzpatrick</strong> [cathyjf]</a> <small>&ndash; Development, Sysadmin</small></p></li>';
 			buf += '<li><p><strong>Hugh Gordon</strong> [V4] <small>&ndash; Research (game mechanics), Development</small></p></li>';
```
Comment (Bryan-0) [inline js/client-mainmenu.js]: <ul> is needed here.

**Comment by Slayer95** [comment]
We should probably move the ``<ul>`` tags to their own lines.
This happens EVERY time.

File: js/client-mainmenu.js (hunk 1 above)
Comment (spekss) [inline js/client-mainmenu.js]: are you sure ? The bug of the screen is in this line there was a `</ul>` at the end, so the lines with V4 and Joim was not in the `<ul>`, but I have add `</ul>` at the end of the joim's line, so all "retired staff" is in the `<ul>` now

**Comment by spekss** [comment]
slayer, do you mean I should put `<ul>` At the beginning of each line? and `</ul>` at the end ?

File: js/client-mainmenu.js (hunk 1 above)
Comment (spekss) [inline js/client-mainmenu.js]: I mean, the `<ul>` is open in the line 1097 and end at the line 1100, all line is in the tag, it's not fine ?

**Comment by Zarel** [comment]
In the long term we need to remove this button entirely...

**Pull request merged**